dojo/models.py

```diff
@@ -457,7 +461,7 @@ class IPScan(models.Model):
 
 
 class Engagement_Type(models.Model):
-    name = models.CharField(max_length=200)
+    name = models.CharField(max_length=200, primary_key=True)
```

Comment:
How would you resolve an issue if there is a duplicate engagement type?

Reply:
I'm not aware if there will be such case. Can you please explain why/when that would happen ?

Reply:
I'm not aware, if there will be such a case. Can you please explain why/when that would happen ?

Reply:
Why change the primary key?

Reply:
@devGregA there is only one field in this model but will make it unique. DRF will throw an error if there is a duplicate engagement type but the bigger question is,  would there be any reason for engagement name to be duplicate  ?